src/Form/Type/PayzenGatewayConfigurationType.php

```diff
@@ -36,6 +36,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'sylius.form.gateway_configuration.payzen.ctx_mode.choices.production' => Api::MODE_PRODUCTION,
                 ),
             ])
+            ->add('endpoint', ChoiceType::class, [
+                'label'       => 'sylius.form.gateway_configuration.payzen.endpoint.label',
+                'choices'  => array(
+                    'sylius.form.gateway_configuration.payzen.endpoint.choices.default' => null,
+                    'sylius.form.gateway_configuration.payzen.endpoint.choices.clic_and_pay' => Api::ENDPOINT_CLICANDPAY,
+                    'sylius.form.gateway_configuration.payzen.endpoint.choices.scellius' => Api::ENDPOINT_SCELLIUS,
+                    'sylius.form.gateway_configuration.payzen.endpoint.choices.systempay' => Api::ENDPOINT_SYSTEMPAY,
```

Comment:
In this list is Missing SogeCommerce's URL: https://sogecommerce.societegenerale.eu/vads-payment/

Reply:
SogeCommerce is not yet implemented in ekyna/PayzenPayum

Reply:
Waiting for this pr to be merged and released : https://github.com/ekyna/PayumPayzen/pull/5

Reply:
I would say go to merge this since ekyna as made a new release